### PR TITLE
Fix bug 132: Propagate hashoncookie and hashoncookiepath settings to …

### PR DESCRIPTION
**Pull request by jdevalk2**
Fixes bug report #132

### CODE REVIEW

**Comment by jdevalk2** [comment]
I should be added to a group with a signed corporate CLA now

File: internal/apis/admin/v1/types.go [hunk 1]
```diff
@@ -125,11 +125,13 @@ type Upstream struct {
...
-	HashOn       string        `json:"hash_on,omitempty"`
-	HashOnHeader string        `json:"hash_on_header,omitempty"`
-	HashFallback string        `json:"hash_fallback,omitempty"`
-	Healthchecks *Healthchecks `json:"healthchecks,omitempty"`
-	Slots        int           `json:"slots,omitempty"`
+	HashOn           string        `json:"hash_on,omitempty"`
+	HashFallback     string        `json:"hash_fallback,omitempty"`
+	HashOnHeader     string        `json:"hash_on_header,omitempty"`
+	HashOnCookie     string        `json:"hash_on_cookie,omitempty"`
+	HashOnCookiePath string        `json:"hash_on_cookie_path,omitempty"`
```
Comment (hbagdi) [inline internal/apis/admin/v1/types.go]: Reading the [documentation](https://docs.konghq.com/0.14.x/admin-api/#add-upstream), it seems like we are also missing `hash_fallback_header` property.
Could you take care of it in this PR?

Comment (jdevalk2) [reply internal/apis/admin/v1/types.go]: I added the hash fallback header config setting propagation as well

**Comment by hbagdi** [comment]
> I am added to a group with a signed corporate CLA now. Is there a bot that checks CLA membership?

We don't check for CLAs as of this moment.

Thank you for contributing the fix!

**Pull request merged**